airrohr-firmware/airrohr-firmware.ino

```diff
@@ -4508,6 +4589,12 @@ void loop(void) {
       		sum_send_time += sendCFA(result_PPD, PPD_API_PIN, FPSTR(SENSORS_PPD42NS), "PPD_");
 			sum_send_time += sendSensorCommunity(result_PPD, PPD_API_PIN, FPSTR(SENSORS_PPD42NS), "PPD_");
 		}
+		if (cfg::rtc_read) {
```

Comment:
@Warukira we don't need to send RTC values as they are only timestamps without any sensor data.

Reply:
Noted @chegejames